flink-connectors/flink-connector-pulsar/src/test/java/org/apache/flink/connector/pulsar/source/reader/deserializer/PulsarDeserializationSchemaTest.java

```diff
@@ -64,27 +59,6 @@ void createFromFlinkDeserializationSchema() throws Exception {
         assertEquals(collector.result, "some-sample-message");
     }
 
-    @Test
-    void createFromPulsarSchema() throws Exception {
```

Comment:
We don't need this test as we are not using `PulsarSchemaWrapper` to really decode any messages